test/box-luatest/box_info_memory_runtime_test.lua

```diff
@@ -0,0 +1,51 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+g.before_each(function()
+    g.server = server:new({alias = 'box_info_memory_runtime'})
+    g.server:start()
+end)
+
+g.after_each(function()
+    g.server:drop()
+end)
+
+local function runtime_memory()
+    return g.server:call('box.info.memory').runtime
+end
+
+local function allocate_tuples()
+    return g.server:eval([[do
```

Comment:
Please use `exec` instead of `eval`. BTW, you can use `require('luatest').assert()` from `exec`, e.g.

https://github.com/tarantool/tarantool/blob/cbc3daed35068ea5412aa1692e2bd5f7e2153f29/test/box-luatest/gh_7005_session_on_commit_rollback_test.lua#L23-L38

This could simplify the test.

Reply:
The idea to wrap the whole test into `:exec()` didn't came into my head. I rewrote it and I agree: it is simpler now. Thanks for the suggestion!